deployment/manifest/network.go

```diff
@@ -79,3 +74,15 @@ func (n Network) Interface(staticIPs []string, networkDefaults []NetworkDefault)
 
 	return networkInterface, nil
 }
+
+func ipMaskString(ipMask net.IPMask) string {
+	ip := net.IP(ipMask)
+
+	if p4 := ip.To4(); len(p4) == net.IPv4len {
+		return ip.String()
+	}
+
+	return fmt.Sprintf("%x:%x:%x:%x:%x:%x:%x:%x",
```

Comment:
Is there a reason we need this custom encoding, rather than being able to rely on `ip.String()` to encode it?